src/workerd/api/crypto/asymmetric.c++

```diff
@@ -563,16 +556,16 @@ private:
     jwk.kty = kj::str("RSA");
     jwk.alg = jwkHashAlgorithmName();
 
-    jwk.n = kj::encodeBase64Url(bigNumToArray(KJ_REQUIRE_NONNULL(rsa.n)));
-    jwk.e = kj::encodeBase64Url(bigNumToArray(KJ_REQUIRE_NONNULL(rsa.e)));
+    jwk.n = kj::encodeBase64Url(KJ_REQUIRE_NONNULL(bignumToArray(KJ_REQUIRE_NONNULL(rsa.n))));
+    jwk.e = kj::encodeBase64Url(KJ_REQUIRE_NONNULL(bignumToArray(KJ_REQUIRE_NONNULL(rsa.e))));
 
     if (getType() == "private"_kj) {
-      jwk.d = kj::encodeBase64Url(bigNumToArray(KJ_REQUIRE_NONNULL(rsa.d)));
-      jwk.p = kj::encodeBase64Url(bigNumToArray(KJ_REQUIRE_NONNULL(rsa.p)));
-      jwk.q = kj::encodeBase64Url(bigNumToArray(KJ_REQUIRE_NONNULL(rsa.q)));
-      jwk.dp = kj::encodeBase64Url(bigNumToArray(KJ_REQUIRE_NONNULL(rsa.dmp1)));
-      jwk.dq = kj::encodeBase64Url(bigNumToArray(KJ_REQUIRE_NONNULL(rsa.dmq1)));
-      jwk.qi = kj::encodeBase64Url(bigNumToArray(KJ_REQUIRE_NONNULL(rsa.iqmp)));
+      jwk.d = kj::encodeBase64Url(KJ_REQUIRE_NONNULL(bignumToArray(KJ_REQUIRE_NONNULL(rsa.d))));
```

Comment:
it didn't throw on invalid parse before, right? Is this important that it now does?

Reply:
I don't think it is going to matter one way or the other. It's exceedingly unlikely this would every throw in this case.